### PR TITLE
Ecosystem blurb

### PR DESCRIPTION
**Pull request by ianm99**
This addresses the prompt "describing where your packages fit into the Python ecosystem (are there any other Python packages that have the same/similar functionality?"

Not sure if this has been put in the right place within the readme?

### CODE REVIEW

File: src/dscigametrics/metric1.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    end_date : int
+        Date when campaign ended
+
+    Returns
+    -------
+
+    string
+    A body of text stating the four computed metrics
+
+    Examples
```
Comment (WeilinHan8) [inline src/dscigametrics/metric1.py]: No need to include examples for now.

File: src/dscigametrics/metric1.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+def compute_metrics(data, campaign, start_date, end_date):
+    '''
```
Comment (WeilinHan8) [inline src/dscigametrics/metric1.py:2]: One blank line before summary

File: src/dscigametrics/metric1.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        2. Conversion rate
+        3. Total transaction revenue
+        4. Average transaction revenue
+
+    Parameters
+    ----------
+
+    data : dataframe
+        Dataframe containing information from google analytics
+    campaign : int or double
```
Comment (WeilinHan8) [inline src/dscigametrics/metric1.py]: just int is okay

**Review by WeilinHan8 — CHANGES_REQUESTED** [review]

File: src/dscigametrics/metric1.py (hunk 2 above)
Comment (ianm99) [reply src/dscigametrics/metric1.py:2]: line inserted~

File: src/dscigametrics/metric1.py (hunk 3 above)
Comment (ianm99) [reply src/dscigametrics/metric1.py]: double deleted

File: src/dscigametrics/metric1.py (hunk 1 above)
Comment (ianm99) [reply src/dscigametrics/metric1.py]: duly removed

**Review by WeilinHan8 — APPROVED** [review]
Looks good!

**Pull request merged**